### ISSUE TITLE
OSE_INSTALL_ADDRESS got blank

### ISSUE THREAD

**Issue by AndreFelipeMachado** [lucascbeyeler/zmbackup#102]
##### ISSUE TYPE
- Bug Report

##### ENVIRONMENT VERSION
- Zmbackup Version: 1.2.2
- Zimbra Version: 8.8.8
- Linux Distribution & Version: ubuntu 14.04 lts

##### SUMMARY
Guess that during installation the ip address got blank because of installScript/vars.sh hardcoded domain instead of using the variable OSE_INSTALL_DOMAIN

```
OSE_INSTALL_DOMAIN=`su -s /bin/bash -c "$OSE_INSTALL_DIR/bin/zmprov gad | head -1" $OSE_USER`                                                  # Zimbra's Domain
OSE_INSTALL_ADDRESS=`ping -c1 zimbra.example.com.br | head -1 | cut -d" " -f3`                                            # Zimbra's Server Address
```

##### STEPS TO REPRODUCE
Fresh install

##### ACTUAL RESULTS
OSE_INSTALL_ADDRESS got blank

**Comment by ananiasfilho** [comment]
Confirm! it occurred on CentOS 7 platfform. Need setup it manual.

**Comment by lucascbeyeler** [comment]
Hello @ananiasfilho and @AndreFelipeMachado,

Sorry for the delay, yes the bug was because I forgot to remove that zimbra.example.com.br. Sorry for that.

Could you guys validate if the bug persist?

Regards,
Lucas Costa Beyeler

**Comment by lucascbeyeler** [comment]
Closing the issue because this bug is already fixed in branch 1.2-version
